service/controller/internal/nodecontroller/controller.go

```diff
@@ -0,0 +1,101 @@
+package nodecontroller
```

Comment:
This is a dynamically created/removed controller which exists for each KVMConfig resource and which watches a WC API for node events (mostly Ready condition).